src/os/ui/layer/vectorstylecontrols.js

```diff
@@ -160,3 +173,23 @@ os.ui.layer.VectorStyleControlsCtrl.prototype.onLineDashChange = function() {
     this.scope.$emit(os.ui.layer.VectorStyleControlsEventType.LINE_DASH_CHANGE, this.scope['lineDash']);
   }
 };
+
+
+/**
+ * Search result formatter. The select is actually storing the ID of each
+ * descriptor. This function allows us to display the actual layer title.
+ * @param {Object} item
+ * @param {angular.JQLite} ele
+ * @return {string|angular.JQLite}
+ * @private
+ */
+os.ui.layer.VectorStyleControlsCtrl.prototype.select2Formatter_ = function(item, ele) {
+  if (item) {
+    var val = '<svg height="2" width="80"><g fill="none" stroke="white" stroke-width="3"><path ';
```

Comment:
I think the hard-coded `stroke` colour and possibly the `d` parameter might be theming problems. For example, if you put OpenSphere into "Light" theme, you can only see the line style for the one you have selected (because the line style and background are both white otherwise). I don't remember what they are though, sorry.

If you were changing the `d`, you could break out the magic numbers as symbolic variables (or constants if applicable).

Reply:
The `d` seems to be okay to keep
I moved the stroke fill to a class so it updates with the theme
Also the search box didn't make sense so that has been removed